inmaputil/cmd.go

```diff
@@ -655,16 +666,23 @@ concentrations with no temporal variability.`,
 		if err != nil {
 			return err
 		}
+
+		shapeFiles := expandStringSlice(Cfg.GetStringSlice("EmissionsShapefiles"))
+		// wrap download over each filepaths
+		for _, element := range shapeFiles {
+			element = download(element, createChan(cmd))
+		}
+
 		return Run(
 			checkLogFile(Cfg.GetString("LogFile"), outputFile),
 			outputFile,
 			Cfg.GetBool("OutputAllLayers"),
 			outputVars,
 			emisUnits,
-			expandStringSlice(Cfg.GetStringSlice("EmissionsShapefiles")),
+			shapeFiles,
 			vgc,
-			os.ExpandEnv(Cfg.GetString("InMAPData")),
-			os.ExpandEnv(Cfg.GetString("VariableGridData")),
+			download(os.ExpandEnv(Cfg.GetString("InMAPData")), createChan(cmd)),
```

Comment:
An issue with having the `createChan` function and then using it over and over again is that it ends up creating a bunch of channels and then never closing them. It might be better to create one channel and then use the same channel for all the download functions, and then close the channel when done with it.